### PR TITLE
Gw models

### PR DESCRIPTION
**Pull request by gretchenmarie**
## Link to Ticket (not a ticket)
https://github.com/NSS-Likeable-Lemurs/BangazonAPI

## Description of Proposed Changes
built out the models for all of the database components 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <Models>
there is not SQL information just adding the models in

```

## Impacted Areas in Application

The following models
Computer
Computer Employee
employee
employee training
order
order product
training program

## Mentions @username

@askingalot 
@Akeen11 
@KaylaReid 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.

### CODE REVIEW

**Review by KaylaReid — APPROVED** [review]
looks good!

File: Bangazon/Models/ComputerEmployee.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (askingalot) [inline Bangazon/Models/ComputerEmployee.cs]: I'm not sure you're going to need this model, but I'm not going to ask you to remove it....you *might* need it :)

Comment (gretchenmarie) [reply Bangazon/Models/ComputerEmployee.cs]: ok

**Review by askingalot — APPROVED** [review]
These look good as a start. Remember, though, that you might need to make changes here and there as you go, and that's fine.

**Pull request merged**